test/userController.js

```diff
@@ -0,0 +1,320 @@
+const chai = require('chai'),
+  dictum = require('dictum.js'),
+  server = require('./../app'),
+  should = chai.should();
+
+describe('user', () => {
+  describe('/user POST', () => {
+    it('Should successfully POST given an user', done => {
+      // Given
+      const newUser = {
+        firstName: 'FirstName',
+        lastName: 'LastName',
+        email: 'test@wolox.com.ar',
+        password: 'passwordTest1'
+      };
+      // When
+      chai
+        .request(server)
+        .post('/users')
+        .send(newUser)
+        .then(res => {
+          // Expect
+          res.should.have.status(200);
+          done();
+        });
+    });
+    it('Should throw an error when sending to POST an user with empty fields', done => {
+      // Given
+      const anUserWithoutEmailAndPassword = {
+        firstName: 'FirstName',
+        lastName: 'LastName'
+      };
+      // When
+      chai
+        .request(server)
+        .post('/users')
+        .send(anUserWithoutEmailAndPassword)
+        .then(() => done())
+        .catch(err => {
+          // Expect
+          err.should.have.status(401);
+          err.response.body.should.be.an('array');
+          err.response.body.length.should.be.eq(2);
+          err.response.body.should.include('Invalid Password');
+          err.response.body.should.include('Invalid Mail');
+          done();
+        });
+    });
+    it("Should throw an error when sending to POST when the user's email already exists", done => {
+      // Given
+      const anUserWithAnExistingEmail = {
```

Comment:
This variable has the same values as the `newUser` one :thinking:  maybe we should define it once in the upper scope and simply call it in each test